src/apis/clients/pickly/index.js

```diff
@@ -7,9 +7,10 @@ const {
 
 const POSTS_CLIENT = createClient(posts.host);
 
-const getAllPosts = () => {
+const getAllPosts = pageNum => {
+  const numOfPosts = 5;
```

Comment:
why would you fix this number here? the aim of this PR is to make things dynamic, right?